params/network_params.go

```diff
@@ -59,3 +59,23 @@ const (
 	// the freezer as the cutoff treshold and by clique as the snapshot trust limit.
 	ImmutabilityThreshold = 90000
 )
+
+// //Quorum
+var QuorumImmutabilityThreshold int
+var IsQuorum = false
```

Comment:
We may define ImmutabilityThreshold as var and directly modify it. Although this means we have to modify this network_params file but we can avoid changes in all other files calling to `params.ImmutabilityThreshold`.

Reply:
Want to keep these two distinct and hence added a new variable.

Reply:
Agree with @vsmk98 here, we should stay as close to upstream as we can, so that we don't have merge conflicts down the line

Reply:
Also, as `QuorumImmutabilityThreshold` and `IsQuorum` are only accessed using functions `GetImmutabilityThreshold()` and `SetQuorumImmutabilityThreshold()`, there is no need for them to be exported.

Reply:
Agree Jitu. Have pushed the code changes.